projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -95,6 +95,32 @@ describe('Profile-tag events', () => {
     });
   });
 });
+describe('login notification', () => {
+  const loginAlias = 'loginNotification';
+  beforeEach(() => {
+    cy.server();
+    cdsHelper.setUpMocks();
+    cy.route('POST', '**/users/current/loginnotification**').as(loginAlias);
+    navigation.visitHomePage({});
```

Comment:
We should stay consistent with the test and add a wait for this visit as you have at the beggining of the test

Reply:
The other wait is waiting for the rest request to something like `/cms/pages?`, whereas here we are just going to the home page. Since we already have the waiting for the cms component, which in memory was a bit more reliable, I have removed the other wait.

Reply:
Also, I noticed that in this visitHomePage was not intercepting profiletag, so I went ahead and add that